test/over_react_test/react_util_test.dart

```diff
@@ -29,6 +29,18 @@ part 'react_util_test.over_react.g.dart';
 /// Main entry point for ReactUtil testing
 main() {
   group('ReactUtil', () {
+    group('render behaves as expected when a UiProps instance is provided', () {
+      test('(UiComponent)', () {
+        final renderedInstance = render(Test());
+        expect(Test(getProps(renderedInstance)).children, isEmpty);
+      });
+
+      test('(UiComponent2)', () {
+        final renderedInstance = render(Test2());
+        expect(Test2(getProps(renderedInstance)).children, isEmpty);
```

Comment:
I verified that this test fails without [the analogous change in `react_util.dart`](https://github.com/Workiva/over_react_test/pull/100/commits/9230eb8cc0637694802b13ddbbe1b131995b83e6#diff-21902410a0d355e65ab9c41ad469ad6cR61)